createRelease.sh

```diff
@@ -44,6 +44,13 @@ STARTING_DIR=`pwd`
 REPO=$1
 REPO_NAME=`basename $REPO`
 
+if [[ $REPO_NAME == *"deck"* ]]; then
```

Comment:
This allows it to automatically determine whether it is compressing a slide deck or an exercise repo, assuming the file naming scheme holds.